Examples/Swift/WaypointConfirmationViewController.swift

```diff
@@ -1,14 +1,14 @@
 import UIKit
 
 protocol WaypointConfirmationViewControllerDelegate: NSObjectProtocol {
-    func confirmationControllerDidConfirm(controller: WaypointConfirmationViewController)
+    func confirmationControllerDidConfirm(_: WaypointConfirmationViewController)
```

Comment:
Let's keep the name but avoid the prefix so that the scope variable comes with a predefined name. (`_ controller`)